skin.json

```diff
@@ -146,6 +146,9 @@
       "height": 81
     }
   },
+  "live": {
+    "dvrEnabled": false
```

Comment:
@aldigun, would you be okay if we renamed the variable to:

forceDvrDisabled

which will allow people to disable DVR if they would like?